apps/frontend/src/hooks/aave/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { utils } from 'ethers';
+
+import { Decimal } from '@sovryn/utils';
+
+export const isBetweenZeroAndOne = (value: Decimal): boolean => {
```

Comment:
Do we really need this one?

Reply:
i've leave it inside the calculateUtilizationRate Just In Case and return Zero in that case.